_includes/site/footer.html

```diff
@@ -1,17 +1,133 @@
-<footer>
-  <div class="row">
-    <div class="col s12 m12 l6 crystal">
-      Crystal is licensed under the Apache License,
-      <a href='https://www.apache.org/licenses/LICENSE-2.0' target='_blank'>
-        Version 2.0
-      </a>
-    </div>
-    <div class="col s12 m12 l6 manas right-align black-text">
-      Crystal language, born & raised at
-      <a href="https://manas.tech" target="_blank" class="black-text">Manas</a>
-      <a href="https://manas.tech" target="_blank" class="logo">
-        <i class="manas"></i>
-      </a>
-    </div>
-  </div>
-</footer>
+<nav class="footer-section" aria-label="Home section links">
```

Comment:
This isn't working well: it's taking the `home` list for all the lists.

Also, would it be over-engineering it to have an outer iteration for the lists, given that everything in each `<nav>` element is the essentially the same?

Reply:
I don't follow what you mean. Could you elaborate?

Yes, I think that would be over-engineered. It makes it harder to move stuff around. A little duplication isn't bad.

Reply:
> It makes it harder to move stuff around.

I don't get how, but OK, it was a question and not a request after all.